src/Parallel-Tests.ps1

```diff
@@ -23,7 +23,14 @@ foreach ($a in $assemblies)
 {
     $xmlName = 'xUnit-Results-' + [System.IO.Path]::GetFileNameWithoutExtension($a) + '.xml'
     $outXml = $(Join-Path $outDir $xmlName)
-    $cmdLine = $xunitRunner.FullName + ' ' + $a + ' ' + $testFilter + ' -xml ' + $outXml + ' -parallel none -noshadow -verbose' 
+	if([System.IO.Path]::GetFileNameWithoutExtension($a) -eq 'DefaultCluster.Tests')
```

Comment:
I believe you could just remove the `-parallel none` switch from here, and instead rely on the <dllname>.xunit.runner.json files to specify by themselves how they parallelize the run for their own test assembly.